.github/workflows/release-please.yml

```
@@ -10,6 +10,7 @@ permissions:
 jobs:
   release-please:
     runs-on: ubuntu-latest
+    if: ${{ github.repository == ‘statnett/controller-runtime-viper’ }}
```

Why the fancy quotes?

```suggestion
    if: ${{ github.repository == 'statnett/controller-runtime-viper' }}
```